<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NIPUNVA01" ignored="0" name="TestNG Test ie" tests="1" failures="1" timestamp="22 Mar 2018 08:42:31 GMT" time="10.838" errors="0">
  <testcase name="TestFlow" time="0.008" classname="amdocs.testing.automation.Level9TestNG.TestNGTest1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at amdocs.testing.automation.Level9TestNG.TestNGTest1.TestFlow(TestNGTest1.java:40)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- TestFlow -->
</testsuite> <!-- TestNG Test ie -->
